src/test/java/com/training/services/tests/GACreateTest.java

```diff
@@ -0,0 +1,73 @@
+package com.training.services.tests;
+
+import com.google.gson.Gson;
+import com.training.base.BaseTest;
+import com.training.services.ga.authenticate.*;
+import com.training.services.ga.create.RequestBodyCreate;
+import com.training.utilities.RestEngine;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.HashMap;
+import java.util.Map;
+import static com.training.utilities.UniqueMailId.getUniqueMailId;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class GACreateTest extends BaseTest {
+    RequestBodyCreate requestBodyCreate;
+    Map<String, Object> headerMap;
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @BeforeAll
+    public  void setData() {
+        headerMap = new HashMap();
+        headerMap.put(map.get("AppKeyHeader"),map.get("AppKeyValue"));
+        headerMap.put(map.get("ContentTypeHeader"), map.get("ContentTypeValue"));
+    }
+
+    /**
+     * testGuestCreation():request body implemented using POJO concept to get the +ve reponse ie 200
+     */
+    @Test
+    public void  testGuestCreation(){
+        requestBodyCreate=new RequestBodyCreate();
+        requestBodyCreate.setBirthdate("19620802");
+        String uid=getUniqueMailId();
+        requestBodyCreate.setEmail(getUniqueMailId());
```

Comment:
use variable uid here, or-else your test can fail when there is a delay between executing these 2 lines